Rocket.Chat/NewRoomViewController.swift

```diff
@@ -169,11 +169,15 @@ class NewRoomViewController: BaseViewController {
                     completion(false, result.error)
                     return
             }
+            guard let name = result.name else {
+                    completion(false, result.error)
+                    return
```

Comment:
I believe the spacing here is wrong. Can you confirm it's 4 spaces?

Reply:
I just copy/pasted the `guard let success` line, so I'll change that too 😄